templates/components/api-server/template/server/middleware.json

```diff
@@ -19,8 +19,14 @@
   "parse": {
   },
   "routes": {
+    "loopback#rest": {
+      "paths": ["${restApiRoot}"]
+    }
   },
   "files": {
+    "serve-static": {
+      "params": "$!../client"
+    }
```

Comment:
I think a SPA client should not be a part of "API Server" template, please remove this middleware. IIRC, `slc loopback:example` is already adding this middleware entry.